transport-native-epoll/src/main/java/io/netty/channel/epoll/AbstractEpollChannel.java

```diff
@@ -266,7 +264,7 @@ protected final int doWriteBytes(ByteBuf buf) throws Exception {
             } else {
                 nioBuf = buf.nioBuffer();
             }
-            for (;;) {
+            for (int i = writeSpinCount; i >= 0; i--) {
```

Comment:
ups... even did it correct here : https://github.com/netty/netty/pull/3401/files#diff-0151984b13fcb94eb32f49bb667a5d29R100 ... will fix it